packs/spell-effects/spell-effect-evolution-surge.json

```diff
@@ -4,7 +4,7 @@
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Evolution Surge].</p>\n<p>Choose one of the following effects:</p><ul><li>Your eidolon gains low-light vision and darkvision.</li><li>Your eidolon gains @UUID[Compendium.pf2e.bestiary-ability-glossary-srd.Item.Scent] as an imprecise sense up to 30 feet.</li><li>Your eidolon can breathe underwater and gains a swim Speed equal to its land Speed or 30 feet, whichever is less. Alternatively, if your eidolon is normally aquatic, it can breathe air and gains a land Speed equal to its swim Speed or 30 feet, whichever is less. Either way, it gains the amphibious trait.</li><li>Your eidolon gains a +20-foot status bonus to its Speed.</li></ul><hr /><p><strong>Heightened (3rd)</strong> Add the following options to the list of effects you can choose:</p><ul><li>Your eidolon becomes Large, instead of its previous size. This increases your eidolon's reach to 10 feet but doesn't change any other statistics for your eidolon. Because of the special link you share, you can ride your eidolon without getting in each other's way. If another creature tries to ride your eidolon, both you and the riding creature each regain only 2 actions at the start of your turns each round, as normal.</li><li>Your eidolon gains a climb Speed equal to its land Speed.</li></ul><hr /><p><strong>Heightened (5th)</strong> Add the options from the 3rd-rank version and the following options to the list of effects you can choose:</p><ul><li>Your eidolon becomes Huge, instead of its previous size. This has the same effects as the 3rd-rank option to become Large, except your eidolon's reach increases to 15 feet.</li><li>Your eidolon gains a fly Speed equal to its Speed.</li></ul>"
```

Comment:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Evolution Surge]</p>\n<p>Choose one of the following effects:</p><ul><li>Your eidolon gains low-light vision and darkvision.</li><li>Your eidolon gains @UUID[Compendium.pf2e.bestiary-ability-glossary-srd.Item.Scent] as an imprecise sense up to 30 feet.</li><li>Your eidolon can breathe underwater and gains a swim Speed equal to its land Speed or 30 feet, whichever is less. Alternatively, if your eidolon is normally aquatic, it can breathe air and gains a land Speed equal to its swim Speed or 30 feet, whichever is less. Either way, it gains the amphibious trait.</li><li>Your eidolon gains a +20-foot status bonus to its Speed.</li></ul><hr /><p><strong>Heightened (3rd)</strong> Add the following options to the list of effects you can choose:</p><ul><li>Your eidolon becomes Large, instead of its previous size. This increases your eidolon's reach to 10 feet but doesn't change any other statistics for your eidolon. Because of the special link you share, you can ride your eidolon without getting in each other's way. If another creature tries to ride your eidolon, both you and the riding creature each regain only 2 actions at the start of your turns each round, as normal.</li><li>Your eidolon gains a climb Speed equal to its land Speed.</li></ul><hr /><p><strong>Heightened (5th)</strong> Add the options from the 3rd-rank version and the following options to the list of effects you can choose:</p><ul><li>Your eidolon becomes Huge, instead of its previous size. This has the same effects as the 3rd-rank option to become Large, except your eidolon's reach increases to 15 feet.</li><li>Your eidolon gains a fly Speed equal to its Speed.</li></ul>"
```

Reply:
This should probably be updated to "you" rather than "your eidolon"?